File: backend/src/main/java/com/safetunnel/backend/vpn/peer/VpnPeerRepository.java
package com.safetunnel.backend.vpn.peer;

import com.safetunnel.backend.users.VpnPeer;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.UUID;

public interface VpnPeerRepository extends JpaRepository<VpnPeer, UUID> {
    VpnPeer findByUserIdAndServerId(UUID userId, UUID serverId);
}

File: backend/src/main/java/com/safetunnel/backend/vpn/peer/VpnPeerRepository.kt
package com.safetunnel.backend.vpn.peer

import com.safetunnel.backend.users.VpnPeer
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface VpnPeerRepository : JpaRepository<VpnPeer, UUID> {
    fun findByUserIdAndServerId(userId: UUID, serverId: UUID): VpnPeer?
}